public/locales/en/translations.json

```diff
@@ -556,8 +556,25 @@
     },
     "flyouts": {
       "new": {
-        "title": "New Package"
+        "title": "New Package",
+        "header": "Upload a package",
+        "description": "Add a package to your solution",
+        "upload": "Upload",
+        "cancel": "Cancel",
+        "close": "Close",
+        "type": "Package type",
+        "browse": "Browse",
+        "browseText": "for a package file",
+        "placeHolder": "Select package type",
+        "package": "Package",
+        "deploymentText":"To deploy packages, go to the 'Deployments' page and click '+ New Deployment' button.",
```

Comment:
Use a param to set the name of the button. This will ensure that the translation is consistent.

Reply:
This is still a work in progress. Once the deployment page is complete, I have to break this into sub groups of words and hyperlink '+ New Deployment' which routes to Deployment page.